apps/back/src/Twig/AppExtension.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+class AppExtension extends AbstractExtension
+{
+    public function getFilters(): array
+    {
+        return array(
+            new TwigFilter('divide_by_100', array($this, 'divideBy100')),
+        );
+    }
+
+    public function divideBy100($amount): float
```

Comment:
J'aurais personnellement parlé de formater la valeur plus que la modifier

```suggestion
    public function formatCurrency($amount): float
```